### ISSUE TITLE
How are Mac users opening mjpeg files?

### ISSUE THREAD

**Issue by hfroot** [HPCE/hpce-2016-cw6#7]
I read that MPlayerX might be able to play them, but I was wondering if anyone had used this or is using something else to avoid downloading a swarm of odd software.

**Comment by jeremych1000** [comment]
VLC?

**Comment by hfroot** [comment]
That lets me see it, however nothing seems to be changing. Should it change?

**Comment by m8pple** [comment]
It should change as long as there is more than one frame -for the default
graph I suggested there should be some movement (it isn't especially pretty though).

If it is static then mostly likely it is displaying it as a single JPEG (as MJPEG is
just a bunch of JPEGs concatenated into a file).

When I was using a mac I used `ffmpeg`, which I've used to play MJPEGs
created in this way before. It comes with a utility called `ffplay` which is
a very barebones player. So:

    ffplay w/heat63.mjpeg

should show the animation.

Updated: Yup, just resurrected my mac. Before it crashed, I compiled cw6 and
ran the default example, and can confirm that ffmpeg shows the video.

 `ffmpeg` is available via ports, and I assume via `brew` as well.

I should point out that the outputs of the simulations are not needed
day-to-day, they are only there to make it more realistic in terms of
output. Because the application is so cut-down, it is difficult to do
real-world physics. Though the mesh graphs are sometimes pretty.
